test/MileageTest.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Mileage.h"
+#include <math.h>
+#include <gtest/gtest.h>
+
+namespace etrobocon2021_test {
+  TEST(calculateMileageTest, calculateMilage)
+  {
+    Mileage& mileage = Mileage::getInstance();
+    double rightWheel = 2 * 10 * 45 * M_PI / 360;
+    double leftWheel = 2 * 20 * 45 * M_PI / 360;
```

Comment:
```suggestion
    double rightWheel = 2 * radius * rightAngle * M_PI / 360;
    double leftWheel = 2 * radius * leftAngle * M_PI / 360;
```
マジックナンバーは使わない方が良いと思われる